src/java/org/apache/cassandra/db/compaction/LeveledManifest.java

```diff
@@ -242,11 +242,17 @@ public synchronized CompactionCandidate getCompactionCandidates()
             // we want to calculate score excluding compacting ones
             Set<SSTableReader> sstablesInLevel = Sets.newHashSet(sstables);
             Set<SSTableReader> remaining = Sets.difference(sstablesInLevel, cfs.getTracker().getCompacting());
-            double score = (double) SSTableReader.getTotalBytes(remaining) / (double)maxBytesForLevel(i, maxSSTableSizeInBytes);
+            long remainingBytesForLevel = SSTableReader.getTotalBytes(remaining);
```

Comment:
Here I rely on the remaining bytes rather than total bytes and I feel it is safe to do so because on the first compaction attempt total = remaining. Please, confirm the assumption is correct.